config/media-library.php

```diff
@@ -198,6 +198,13 @@
      */
     'media_downloader' => Spatie\MediaLibrary\Downloaders\DefaultDownloader::class,
 
+    /*
+     * When using the addMediaFromUrl method the SSL is verified by default.
+     * This is option disables SSL verification when downloading remote media locally.
```

Comment:
nitpick: can we remove the word "locally" from "when downloading remote media locally"? Technically speaking, it disables SSL regardless of whether it's a local URL.

Reply:
Good call! Removed that word.